### DISCUSSION TITLE
Global menu with libdbusmenu

### DISCUSSION THREAD

**Discussion by paulcarroty** [VSCodium/vscodium#1458] [Ideas]
Can we add the libdbusmenu dependency to build environment? @daiyam
Moved from [here](https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/-/issues/91).

**Comment by daiyam** [comment]
Won't that messed up other distros? Seems to be generics enough.

**Comment by paulcarroty** [comment]
This library is popular, seems like: https://repology.org/project/libdbusmenu/versions
If it will not cause conflicts with non-KDE environments, will be nice to have.

**Reply by daiyam** [reply]
I will add it to the insiders version.
